src/components/record-report-page/record-report-list/record-item/index.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { memo, Fragment } from 'react';
+import get from 'lodash/get';
+
+import SC from './styled';
+import { localization } from '../../../../lib/localization';
+import { Record } from '../../../../types';
+import { RecordHeader } from './record-header';
+import { ContactInformation } from '../../contact-information';
+
+interface Props {
+  record: Record;
+}
+
+export const RecordItem = memo(({ record }: Props) => (
+  <>
+    <RecordHeader title={record.title} />
+
+    <SC.Section>
+      <SC.Title>{localization.dataProcessor}</SC.Title>
+
+      <SC.SectionColumn>
+        <SC.SectionContent>
+          <SC.SubTitle>{localization.commonDataControllerContact}</SC.SubTitle>
+
+          {get(record, ['commonDataControllerContact', 'companies']) && (
+            <div>
+              <span>{record.commonDataControllerContact.companies}</span>
+            </div>
+          )}
+          {get(record, [
+            'commonDataControllerContact',
+            'distributionOfResponsibilities'
+          ]) && (
+            <div>
+              <span>
+                {
+                  record.commonDataControllerContact
+                    .distributionOfResponsibilities
+                }
+              </span>
+            </div>
+          )}
+        </SC.SectionContent>
+
+        <SC.SectionContent>
+          <SC.SubTitle>{localization.dataProcessorContactDetails}</SC.SubTitle>
+          {record.dataProcessorContactDetails && (
+            <ContactInformation
```

Comment:
Unødvendig abstraksjon

Reply:
'ContactInformation' skal gjenbrukes også for representatives som kommer i ny PR.